installer/airgapped/install_keptn.sh

```diff
@@ -22,24 +22,39 @@ kubectl create namespace "${KEPTN_NAMESPACE}"
-control-plane.distributor.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/distributor,\
-control-plane.shipyardController.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/shipyard-controller,\
-control-plane.configurationService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/configuration-service,\
-control-plane.mongodbDatastore.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/mongodb-datastore,\
-control-plane.statisticsService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/statistics-service,\
-control-plane.lighthouseService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/lighthouse-service,\
-control-plane.secretService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/secret-service,\
-control-plane.approvalService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/approval-service,\
-continuous-delivery.distributor.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY}keptn/distributor"
+control-plane.remediationService.image.registry=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.remediationService.image.repositoryImageName=keptn/remediation-service,\
+control-plane.apiService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.apiService.image.repositoryImageName=keptn/api,\
+control-plane.bridge.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.bridge.image.repositoryImageName=keptn/bridge2,\
+control-plane.distributor.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.distributor.image.repositoryImageName=keptn/distributor,\
+control-plane.shipyardController.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.shipyardController.image.repositoryImageName=keptn/shipyard-controller,\
+control-plane.configurationService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.configurationService.image.repositoryImageName=keptn/configuration-service,\
+control-plane.mongodbDatastore.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.mongodbDatastore.image.repositoryImageName=keptn/mongodb-datastore,\
+control-plane.statisticsService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.statisticsService.image.repositoryImageName=keptn/statistics-service,\
+control-plane.lighthouseService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.lighthouseService.image.repositoryImageName=keptn/lighthouse-service,\
+control-plane.secretService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.secretService.image.repositoryImageName=keptn/secret-service,\
+control-plane.approvalService.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+control-plane.approvalService.image.repositoryImageName=keptn/approval-service,\
+continuous-delivery.distributor.image.repository=${TARGET_INTERNAL_DOCKER_REGISTRY},\
+continuous-delivery.distributor.image.repositoryImageName=keptn/distributor"
```

Comment:
Setting a different registry shouldn't be this painful.
Think of the users.

Setting a different registry is far simpler in other charts: it can be a [global value](https://helm.sh/docs/chart_template_guide/subcharts_and_globals/#helm).
For the most part it is/was here, too.

```shellsession
LOM0227@WKMJ0E2978:~/code/keptn/installer/manifests$ git checkout 754762464ead3a38acbe8cdd1499f99e770fa7bb
HEAD is now at 75476246 fix(distributor): Ensure that the subscriptionID is passed to the event (#5405) (#5412)
LOM0227@WKMJ0E2978:~/code/keptn/installer/manifests$ helm template keptn ./keptn | grep image:
        - image: docker.io/centos/mongodb-36-centos7:1
          image: docker.io/nginxinc/nginx-unprivileged:1.19.4-alpine
        image: docker.io/keptn/remediation-service:latest
        image: docker.io/keptn/distributor:latest
          image: docker.io/keptn/api:latest
          image: docker.io/keptn/distributor:latest
          image: docker.io/keptn/bridge2:latest
          image: docker.io/keptn/shipyard-controller:latest
          image: docker.io/keptn/distributor:latest
          image: docker.io/keptn/secret-service:latest
        image: docker.io/alpine:3.13
          image: docker.io/keptn/configuration-service:latest
          image: docker.io/keptn/statistics-service:latest
          image: docker.io/keptn/distributor:latest
          image: docker.io/keptn/approval-service:latest
          image: docker.io/keptn/distributor:latest
          image: docker.io/keptn/webhook-service:latest
          image: docker.io/keptn/distributor:latest
        image: docker.io/keptn/mongodb-datastore:latest
        image: docker.io/keptn/distributor:latest
          image: docker.io/keptn/lighthouse-service:latest
          image: docker.io/keptn/distributor:latest
LOM0227@WKMJ0E2978:~/code/keptn/installer/manifests$ helm template --set global.imageRegistry=registry keptn ./keptn | grep image:
        - image: registry/centos/mongodb-36-centos7:1
          image: registry/nginxinc/nginx-unprivileged:1.19.4-alpine
        image: registry/keptn/remediation-service:latest
        image: registry/keptn/distributor:latest
          image: registry/keptn/api:latest
          image: registry/keptn/distributor:latest
          image: registry/keptn/bridge2:latest
          image: registry/keptn/shipyard-controller:latest
          image: registry/keptn/distributor:latest
          image: registry/keptn/secret-service:latest
        image: registry/alpine:3.13
          image: registry/keptn/configuration-service:latest
          image: registry/keptn/statistics-service:latest
          image: registry/keptn/distributor:latest
          image: registry/keptn/approval-service:latest
          image: registry/keptn/distributor:latest
          image: registry/keptn/webhook-service:latest
          image: registry/keptn/distributor:latest
        image: registry/keptn/mongodb-datastore:latest
        image: registry/keptn/distributor:latest
          image: registry/keptn/lighthouse-service:latest
          image: registry/keptn/distributor:latest
```
Though it probably should have been documented somewhere, I never got that feedback in pull request.

While we can manage dysfunction in our charts, dysfunction in external charts remain, so those will need values they accept.
```yaml
control-plane:
  nats:
    nats:
      image: registry/nats:2.3.4-alpine
    bootconfig:
      image: registry/natsio/nats-boot-config:0.5.3
    natsbox:
      image: registry/natsio/nats-box:0.6.0
    reloader:
      image: registry/natsio/nats-server-config-reloader:0.6.1
    exporter:
      image: registry/natsio/prometheus-nats-exporter:0.8.0
```
I suggest going to their repos & requesting they implement configurable registries, too.

Before writing a script like this, someone should explore better solutions/root cause & resolve it there.